src/borg/helpers.py

```diff
@@ -1546,3 +1547,27 @@ def signal_handler(signo, handler):
         yield
     finally:
         signal.signal(signo, old_signal_handler)
+
+
+class ErrorFreeTextIOWrapper(io.TextIOWrapper):
+    def read(self, n):
+        if not self.closed:
+            try:
+                return super(ErrorFreeTextIOWrapper, self).read(n)
+            except IOError:
```

Comment:
shall we maybe ignore a bit more specifically?
like only the closed pipe error?